### ISSUE TITLE
Recipe Style Guide & Repo Structure

### ISSUE THREAD

**Issue by gitgrubber** [git-grub/General#1]
Obviously Markdown has its own intrinsic style guide, with the exception of there being multiple formats for things like titles ( # vs == ). I think we should establish a style for writing recipes, however.

As for structure I've attempted to lay down the first step by creating a country-specific repo. I'm not sure this is right- though- since one might wish to explore by food type.

Dear me, this is what happens when programmers approach recipe curation!

**Comment by ElectronicsGeek** [comment]
My thinking is that we should have a repo per recipe, forked off a recipe template repository (to keep things consistent). By having a recipe per repository, users can fork a single recipe more easily to add their own changes.

Additionally, each repo could have a "gh-pages" branch which would enable the viewing and even printing of a recipe outside of github.

After a recipe has been created, its GH pages URL could be added to the main website (git-grub.github.io).

Just an idea. Thoughts?

**Comment by tyrower** [comment]
Sounds good to me particularly standardised templates and serving up the GH webpages. I'm only learning the Github ropes at the moment but this is a good opportunity to get to grips with the platform. 

I have 2 recipes I'm hoping to (re)try out and further develop this week. Most of mine will be French/Irish mix. Suppose we could find a way to index and cross reference as we go along. Any way of adding "Keywords" to Git repos? Thinking with country/type/main ingredient/course could be an easy way to search/browse recipes.
Voilà - a few of my first thoughts. Bon appetit!  {Eating this evening reckoned could even add my basic vinaigrette/salad dressing recipe...}

**Comment by ElectronicsGeek** [comment]
If I have any time tomorrow, I'll get to work on the base repo.

**Comment by Gadgetoid** [comment]
This makes good sense to me- specially using a standard recipe template repo. Indexing is definitely an option- we've got some code at Pim for turning Markdown from GitHub into a complete, searchable and faceted website. It's uses my concept of "YAML tagged markdown" which is markdown with commented out Yaml on top for adding metadata. If we have a repo per recipe however then we can easily do this with a separate Yaml file.

**Comment by ElectronicsGeek** [comment]
Incidentally, that's exactly how Jekyll works (YAML topped markdown)!

**Comment by Gadgetoid** [comment]
Is that what Jekll is? I had no idea. I might need to look into it. Haha. Clearly it's not as crazy as solution as I'd thought.

**Comment by ElectronicsGeek** [comment]
Yeah. It's a content management system for static webpages. You can use it on GitHub pages, which is why I suggested it 😀
